### PR TITLE
[Tablets Orders] Collapsing the drawer when scrolled outside

### LINKED ISSUE #10492: [Tablets Orders] Consider collapsing the drawer when clicked outside if expanded

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #10492

### Description

The PR adds auto collapsing of the bottom drawer when a user scrolls to the bottom of the order creation/editing screen

### Testing instructions

Try on tablets/phones
* Orders -> Order creation
* Add products or anything that makes the scroll view big
* Expand the bottom drawer
* Scroll bottom, notice that it collapses and the whole content of the view is still accessible

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/4923871/54a53ee4-b7ea-4fde-9858-38f60dcef9a7

https://github.com/woocommerce/woocommerce-android/assets/4923871/2db20a7e-cd3b-4260-9f0a-a04647ef6ca0

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">View files have been modified, but no screenshot is included in the pull request. Consider adding some for clarity.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,4 +1,14 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
+17.3
+-----
+- [*] [Internal] Collapse bottom drawer with totals on the order creation/editing screen when scrolling down [https://github.com/woocommerce/woocommerce-android/pull/10573]
+- [*] [Internal] Better displaying of the "locked" state of the buttons on the order creation screen. Fixed bug when it was possible to add "custom amount" even when the order is locked [https://github.com/woocommerce/woocommerce-android/pull/10562]
```
Comment (kidinov) [inline RELEASE-NOTES.txt]: 2bc43e4a30b2d84e2689f5b01f23c5601fd8d150  - I don't know what happened here, but we lost some release notes?

@spencertransier

**Comment by backwardstruck** [comment]
Tablet (Pixel Tablet API 34 emulator):

Screenshot_20240123_160244
Screenshot_20240123_160254

Phone (Pixel 6 Pro API 33 emulator):

Screenshot_20240123_155221
Screenshot_20240123_155145

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (backwardstruck) [reply RELEASE-NOTES.txt]: Was there some merge conflict earlier?

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditFormFragment.kt [hunk 2]
```diff
@@ -1023,6 +1021,11 @@ class OrderCreateEditFormFragment :
                 )
             }
 
+            is OnTotalsSectionHeightChanged -> {
+                println("Totals section height changed: ${event.newHeight}")
```
Comment (backwardstruck) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditFormFragment.kt]: It looks like this `println()` may have just been for testing?

**Review by backwardstruck — COMMENTED** [review]
Tested. Works fine. Just one non-blocking code question.

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (kidinov) [reply RELEASE-NOTES.txt]: It feels like @spencertransier moved some release notes from 17.3 to 17.0, I am not sure if this was deliberate or not. I DM him - let see what will be an answer

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditFormFragment.kt (hunk 2 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/OrderCreateEditFormFragment.kt]: Thanks, good catch! Removed that here - d84ce0e48ec82c4070be222f405839f17c2763aa

**Review by backwardstruck — APPROVED** [review]

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (kidinov) [reply RELEASE-NOTES.txt]: Haven't got any answer, so the PR missed the milestone. Moved it to another one (17.4) and moved release note

Comment (kidinov) [reply RELEASE-NOTES.txt]: Ok so the actual release version we are targeting now is 

PaaHJt-eZ-p2 

So I fixed the release notes and merging now

**Pull request merged**